### PR TITLE
Improving ES installation process

### PR DESCRIPTION
**Pull request by ewolinetz**

### CODE REVIEW

**Comment by ewolinetz** [comment]
@jcantrill @sosiouxme PTAL, reducing layers in ES install and making the ACL seeding smarter by checking if ES is up instead of just `sleep`ing

File: elasticsearch/Dockerfile [hunk 1]
```diff
@@ -21,20 +21,17 @@ RUN rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch && \
   yum install -y -- setopt=tsflags=nodocs \
   java-1.8.0-openjdk \
   elasticsearch && \
```
Comment (sosiouxme) [inline elasticsearch/Dockerfile]: add in `yum clean all` here so the downloaded RPMs are removed from the layer

**Comment by sosiouxme** [comment]
Otherwise, LGTM!

**Comment by ewolinetz** [comment]
added `yum clean all` to clean up after yum install completes

**Pull request merged**